azkaban-common/src/test/java/azkaban/executor/JdbcExecutorLoaderTest.java

```diff
@@ -873,6 +874,29 @@ public void testFetchActiveFlowByExecId() throws Exception {
     Assert.assertEquals(flow1.getVersion(), execFlow1.getVersion());
   }
 
+  @Test
+  public void testFetchRecentlyFinishedFlows() throws Exception {
+    if (!isTestSetup()) {
+      return;
+    }
+
+    ExecutorLoader loader = createLoader();
+    ExecutableFlow flow1 = TestUtils.createExecutableFlow("exectest1", "exec1");
+    loader.uploadExecutableFlow(flow1);
+    flow1.setStatus(Status.SUCCEEDED);
+    flow1.setEndTime(System.currentTimeMillis());
+    loader.updateExecutableFlow(flow1);
+    List<ExecutableFlow> flows = loader.fetchRecentlyFinishedFlows(recentlyFinishedLifetimeMs);
+    Assert.assertEquals(1, flows.size());
+    Assert.assertEquals(flow1.getExecutionId(), flows.get(0).getExecutionId());
+    Assert.assertEquals(flow1.getProjectName(), flows.get(0).getProjectName());
+    Assert.assertEquals(flow1.getFlowId(), flows.get(0).getFlowId());
+    Assert.assertEquals(flow1.getVersion(), flows.get(0).getVersion());
+    Thread.currentThread().sleep(recentlyFinishedLifetimeMs);
```

Comment:
Why does the thread need to sleep here? 
It's important to keep a unit test run as fast as possible. in a few ms is the norm.